src/main/java/org/ambraproject/wombat/service/PeerReviewServiceImpl.java

```diff
@@ -0,0 +1,175 @@
+package org.ambraproject.wombat.service;
+
+import com.google.common.collect.ImmutableList;
+import org.ambraproject.wombat.service.remote.ContentKey;
+import org.ambraproject.wombat.service.remote.CorpusContentApi;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.util.EntityUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.xml.sax.InputSource;
+import org.xml.sax.SAXException;
+import org.xml.sax.XMLReader;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerException;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.sax.SAXSource;
+import javax.xml.transform.stream.StreamResult;
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.xpath.XPath;
+import javax.xml.xpath.XPathExpressionException;
+import javax.xml.xpath.XPathFactory;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeMap;
+import java.util.UUID;
+
+
+public class PeerReviewServiceImpl implements PeerReviewService {
+
+  public static final String DEFAULT_PEER_REVIEW_XSL = "peer-review-transform.xsl";
+
+  @Autowired
+  private CorpusContentApi corpusContentApi;
+
+  public String asHtml(Map<String, ?> itemTable) throws IOException {
+    String xml = getAllReviewsXml(itemTable);
+    String html = xmlToHtml(xml, DEFAULT_PEER_REVIEW_XSL);
+    return html;
+  }
+
+  /**
+   * Convert peer review XML to HTML
+   *
+   * @param allReviewsXml
+   * @return an HTML representation of peer review content
+   */
+  String xmlToHtml(String allReviewsXml, String xsl) {
+    if (allReviewsXml == null) return null;
```

Comment:
should probably update guard condition to return null if either xml or xsl is null. should throw an exception instead?